src/downloader/DownloadCore.java

```diff
@@ -13,8 +15,19 @@
 import java.util.zip.ZipEntry;
 import java.util.zip.ZipFile;
 
+import com.dropbox.core.DbxException;
+import com.dropbox.core.DbxRequestConfig;
+import com.dropbox.core.v2.DbxClientV2;
+import com.dropbox.core.v2.files.FileMetadata;
+
+import com.dropbox.core.v2.users.FullAccount;
+
 public class DownloadCore {
 	List<String> fileList;
+	//Default download mode Dropbox
+	private String mode = "D";
+	// Used for dropbox API
+	private static final String ACCESS_TOKEN = "1rhx1CwxQt8AAAAAAABAWW9BYue31WkKZ2dan-lc2uaC50rjrDZe8vgVksNXrH6v";
```

Comment:
This access token is account specific, I would recommend to add it in the configuration file.